CHANGELOG.md

```diff
@@ -1,219 +1,274 @@
 
-## 2016-04-06 Release 1.2.2
+### Changed
 
-- Fix: puppet apply works again [#105](https://github.com/chocolatey/puppet-chocolatey/issues/105).
 - `$::chocolateyversion` fact is optional - see [#110](https://github.com/chocolatey/puppet-chocolatey/issues/110)
-- Fix: Implement PowerShell Redirection Fix for Windows 2008 / PowerShell v2 - see [#119](https://github.com/chocolatey/puppet-chocolatey/issues/119)
 
+### Fixed
+
+- puppet apply works again [#105](https://github.com/chocolatey/puppet-chocolatey/issues/105).
+- Implement PowerShell Redirection Fix for Windows 2008 / PowerShell v2 - see [#119](https://github.com/chocolatey/puppet-chocolatey/issues/119)
+
+## [1.2.1] - 2015-12-08
+
+### Added
+
+- Support for newer versions of PE
 
-## 2015-12-08 Release 1.2.1
+## [1.2.0] - 2015-11-03
 
-- Small release for support of newer PE versions.
+### Added
 
+- holdable ([#95](https://github.com/chocolatey/puppet-chocolatey/issues/95))
 
-##2015-11-03 Release 1.2.0
+### Fixed
 
-- Implement holdable ([#95](https://github.com/chocolatey/puppet-chocolatey/issues/95))
-- Fix - Use install unless version specified in install ([#71](https://github.com/chocolatey/puppet-chocolatey/issues/71))
+- Use install unless version specified in install ([#71](https://github.com/chocolatey/puppet-chocolatey/issues/71))
 
+## [1.1.2] - 2015-10-02
 
-## 2015-10-02 Release 1.1.2
+### Fixed
 
 - Ensure 0.9.9.9 compatibility ([#94](https://github.com/chocolatey/puppet-chocolatey/issues/94))
-- Fix - Mixed stale environment variables of existing choco install causing issues ([#86](https://github.com/chocolatey/puppet-chocolatey/issues/86))
+- Mixed stale environment variables of existing choco install causing issues ([#86](https://github.com/chocolatey/puppet-chocolatey/issues/86))
 - Upgrade From POSH Version of Chocolatey Fails from Puppet ([#60](https://github.com/chocolatey/puppet-chocolatey/issues/60))
 
+## [1.1.1] - 2015-09-25
 
-## 2015-09-25 Release 1.1.1
+### Added
 
 - Add log_output for chocolatey bootstrap installer script
-- Ensure bootstrap enforces chocolatey.nupkg in libs folder
 - Allow file location for installing nupkg file.
 
+### Changed
 
-## 2015-09-09 Release 1.1.0
-
-- Install Chocolatey itself / ensure Chocolatey is installed (PUP-1691)
-- Adds custom facts for chocolateyversion and choco_install_path
+- Ensure bootstrap enforces chocolatey.nupkg in libs folder
 
+### [1.1.0] - 2015-09-09
 
-## 2015-07-23 Release 1.0.2
+- Install Chocolatey itself / ensure Chocolatey is installed ([PUP-1691](https://tickets.puppetlabs.com/browse/PUP-1691))
+- Custom facts for chocolateyversion and choco_install_path
 
-- Fixes [#71](https://github.com/chocolatey/puppet-chocolatey/issues/71) - Allow `ensure => $version` to work with already installed packages
+## [1.0.2] - 2015-07-23
 
+### Fixed
 
-## 2015-07-01 Release 1.0.1
+- Allow `ensure => $version` to work with already installed packages [#71](https://github.com/chocolatey/puppet-chocolatey/issues/71)
```

Comment:
Here you use code phrase (`), but not in other places. What is the KAC standard? Either way this should be consistent.